purebred.cabal

```diff
@@ -65,7 +65,7 @@ library
                      , Purebred
   build-depends:       base >= 4.9 && < 5
                      , lens
-                     , brick >= 0.20
+                     , brick >= 0.34
```

Comment:
Big jump.  Are we actually using stuff that requires 0.34?  We should only
tighten version bounds when there's actually an issue with versions  outside the bounds.

Reply:
Yeah I want to use the forms introduced by 0.32. Arguably it should be 0.32 then...

Reply:
OK, no worries.  Make a note of that in the commit msg.